### DISCUSSION TITLE
Editable Columns

### DISCUSSION THREAD

**Discussion by backpack-operations** [Laravel-Backpack/community-forum#1046] [Q&A (Help)] [answered]
Hello team,

I just received this on email:

"Hello, how can I use select2 in editable columns.
I need to have an option to search for customers but with editable select it does not allow me to work as I need."

**Comment by pxpm** [comment, marked as answer]
Hello @backpack-operations 

We are in the process of working on that feature. At the moment it's not available yet. 

https://github.com/Laravel-Backpack/editable-columns/issues/95

@jcastroa87 started the work on that but I am not sure what's the status on it. I will ping him about it 🙏 

Cheers
